influxdb-meta/README-short.txt

```diff
@@ -0,0 +1 @@
+InfluxDB Enterprise is a highly available time series database.
```

Comment:
InfluxDB Enterprise is the commercial version of the open source InfluxDB, offering high availability and horizontal scalability.